MediaBrowser.Providers/MediaBrowser.Providers.csproj

```diff
@@ -22,6 +22,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <LangVersion>preview</LangVersion>
```

Comment:
why?

Reply:
Needed for discards in lambda expressions, but the build still fails for some reason

Reply:
Looked into it more and lambda discards are not yet a language feature, even in preview: https://github.com/jellyfin/jellyfin/pull/2447#discussion_r399767135

What I have done in the past is name the first parameter `_`, the second `__` and so on, which isn't great. It might be best to just use normal variable names for now and rely on an analyzer to warn us about them in the future once the language feature is released.

Reply:
Can we revert this change then?